app/views/users/index.html.erb

```diff
@@ -2,6 +2,15 @@
 <p><%= @placeholder_text %></p>
 <ul>
   <% @users.each do |user| %>
-    <li><%= link_to user.name, user_path(user) %></li>
+    <li>
+      <%= link_to user_path(user) do %>
+        <%= user.name %>
+        <% if user.photo.present? %>
+          <img src="<%= user.photo %>" alt="User Photo" width="100px">
+        <% else %>
+          <p>No photo available</p>
+        <% end %>

```

Comment:
- [ ] Kindly add basic styling to match the wireframes. Your pages do not match the wireframes.